hawc/apps/mgmt/templates/mgmt/assessment_details.html

```diff
@@ -1,16 +1,105 @@
 {% extends 'assessment-rooted.html' %}
 
 {% block content %}
-  <h2 class="d-inline-block">Task assignments</h2>
-  <div class="btn-group float-right">
-      <a class="btn btn-primary dropdown-toggle" data-toggle="dropdown">Actions</a>
-      <div class="dropdown-menu dropdown-menu-right">
-          <a class="dropdown-item" href="{% url 'mgmt:user_assessment_assignments' assessment.pk %}">My assigned tasks</a>
-      </div>
+<h2 class="d-inline-block">Task assignments</h2>
+<div class="btn-group float-right">
+  <a class="btn btn-primary dropdown-toggle" data-toggle="dropdown">Actions</a>
+  <div class="dropdown-menu dropdown-menu-right">
+    <a class="dropdown-item" href="{% url 'mgmt:user_assessment_assignments' assessment.pk %}">My assigned tasks</a>
   </div>
-  <div id='main' class='mb-3'></div>
-{% endblock %}
+</div>
+
+<!-- filter by study type section not wired up  -->
```

Comment:
add `TODO` to the comment so we know this needs to be done, and we can search the codebase to find it

Reply:
added `TODO` to comment. Is this something you would like me to approach? If so how?